### DISCUSSION TITLE
Sunbird Installation issues as per given instructions "Sunbird Infra Requirements" in the Diksha tender.

### DISCUSSION THREAD

**Discussion by shounakacharya** [Sunbird-Ed/Community#76] [Installation] [answered]
Hi Sunbird/Diksha Team,

As per DIC RFP for " DIKSHA Platform" , Bidder will have to install the Sunbird platform based on the given installation instructions : http://docs.sunbird.org/latest/developer-docs/server-installation/
But there are many errors encountered while installing the sunbird Platform , We need urgent help from Sunbird/Diksha Team who can sort the issues. We have attached the errors Document, You can see there are some application binaries / components which are required to run the complete sunbird application are either failing during builds or during deployment phase, again by using sunbird automation jobs.

Thus to troubleshoot those we need the sunbird application team, as it is a complete infrastructure as a code deployment & hence we may not be able to solve the issues and challenges arising out of source code.

1.	KP, DP, DB, Yarn, Jenkins, Other-Services: Ubuntu 18.04 LTS, 4vCPU, 16GB RAM, 250 GB HDD
2.	Google Kubernetes engine: 1.21.12-gke.2200 with 4 worker nodes. Note: Currently Sunbird is not compatible with K8s APIs  >= 1.22 and we have checked this
3.	All systems are running on GCP
[sunbird-install-error-document.docx](https://github.com/Sunbird-Ed/Community/files/9395618/sunbird-install-error-document.docx)

@keshavprasadms and @gandham-santhosh

**Comment by beepdot** [comment, marked as answer]
@sharathkashyap Please close this thread. These are being answered in other threads.

**Reply by sharathkashyap** [reply]
@keshavprasadms Closing this thread now
